test/parallel_api/dynamic_selection/sycl/test_auto_tune_policy_sycl.pass.cpp

```diff
@@ -468,59 +466,59 @@ main()
         constexpr bool just_call_submit = false;
         constexpr bool call_select_before_submit = true;
 
-        auto actual = test_auto_initialization(u1);
```

Comment:
It seems like instead of removing `actual`, we should change it to be `int failures`, and adjust the subsequent assignments `+=`, and then check that it is zero at the end.  The function seems to return 1 on failure and 0 on pass.

Reply:
I added EXPECT_EQ: it will report a particular line.